src/main/java/radiounit_executorandreceiver/RadioCommandExecutorFactory.java

```diff
@@ -0,0 +1,12 @@
+package radiounit_executorandreceiver;
+
+/**
+ * Abstract class for RadioCommandExecutorFactory classes
+ * @author esiumat
+ *
+ */
+public abstract class RadioCommandExecutorFactory {
```

Comment:
I can't recall if there was a specific reason we're using an abstract class here (instead of an interface), but based on the fact that we're just exposing a single public method, we could probably get away with an interface - we can implement many interfaces, but only inherit from one abstract class

Reply:
Yeah, I think this is true. I defaulted to abstract since it was the class used in the refactoring guru factory UML. 

I see the benefit of your suggestion. I'll change it.

Reply:
I suppose similarly our RadioUnitReceiverFactory also only exposes one public method. I'll adjust that to an interface as well.